File: Animal.java
package com.company;

public class Animal {
     public final String species = "canis";
     private Double weight = 3.0;
     public String name;

    void printWeight(){
        System.out.println(weight);
    }

    void feed (Double foodWeight){
        weight += foodWeight;
        System.out.println("thx for food, bro");
        System.out.println("my weight is now" + weight );
    }

    String returnSpeciesAndName(){
        return species + " " + name;
    }

    String returnNameAndOwner(String owner){
        return name + " " + owner;
    }
}

File: Animal.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace OOPlab1 {
    public class Animal {
        public readonly string Species = "canis";
        public string Name;
        private double weight = 3.0;

        public void PrintWeight() {
            Console.WriteLine(weight);
        }

        public void Feed(double foodWeight) {
            weight += foodWeight;
            Console.WriteLine("thx for food, bro");
            Console.WriteLine("my weight is now " + weight);
        }

        public string ReturnSpeciesAndName() {
            return Species + " " + Name;
        }

        public string ReturnNameAndOwner(string owner) {
            return Name + " " + owner;
        }
    }
}
